scripts/install/opt/seagate/cortx/motr/common/m0_utils_common.sh

```diff
@@ -613,7 +613,7 @@ spiel_prepare() {
 
         prof_fid_str=$(_create_fid_list "$PROFILE_FID")
         pool_fid_str=$(_create_fid_list "$POOL_FID")
-        SPIEL_FIDS_LIST="fids = {'profile' : $prof_fid_str, 'pool' : $pool_fid_str,}"
+        SPIEL_FIDS_LIST="fids = {'profile' : $prof_fid_str, 'pool' : $pool_fid_str}"
```

Comment:
Isn't comma at the end of dictionary, a valid  syntax?

Reply:
You are right. The SPIEL_FIDS_LIST is processed as a python dictionary. My mistake!
I will take a re-look at the code flow. Thanks!